pgcli/pgexecute.py

```diff
@@ -289,7 +289,9 @@ def execute_normal_sql(self, split_sql):
         cur.execute(split_sql)
 
         try:
-            title = self.conn.notices.pop()
+            title = ''
+            while len(self.conn.notices) > 0:
+                title = title + self.conn.notices.pop()
```

Comment:
I now remember why I was using `pop`, because we're calling notices on the connection object and not the cursor object. So the notices are kept for the whole session unless we clear them. 

I believe your code does the right thing. My apologies.